templates/zerver/for/open-source.md

```diff
@@ -0,0 +1,99 @@
+A great open source project needs a great group chat service.  Here's
+why we think Zulip is the best choice you can make today.
+
+(Psst, I'll spoil the ending: not only is Zulip itself open source,
+but hosting on zulipchat.com is free for open source projects.
+[Sign up now](https://zulipchat.com/beta/)!)
+
+## Great for developers
+
+* Messages take Markdown formatting, [including code
+  blocks](/help/format-your-message-using-markdown#code).  Super easy
+  to discuss code, paste error messages, etc.
+
+* Want to refer to issues or code reviews with short notation like
```

Comment:
Swap this and the next paragraph, as regex filters are somewhat more of an obscure use case than perma-linking.